tests/acceptance/01_vars/02_functions/string_replace.cf

```diff
@@ -0,0 +1,49 @@
+
+#######################################################
+#
+# Test string_replace function
+#
+#######################################################
+
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent test
+{
+  vars:
+      "test" string => "abcdefghij\t\n";
+      "test2" string => "(){}[].*?";
+
+      # normal tests
+      "replace" string => string_replace($(test), "\t", "\r");
+      "replace2" string => string_replace($(test), "ghij\t", "tEsT");
+      "replace3" string => string_replace($(test), "abcdefghij\t\n", "no");
+
```

Comment:
What happens if there is an overlap?

```
    "being_stupid" string => string_replace("aaaa", "aaa", "hello");
```

I hope it skips what has already been matched and results in `"helloa"`?